app/client/templates/about-us.template.html

```diff
@@ -1,9 +1,9 @@
 <article class='wrapper_about_us_section'>
   <article>
     <section class='summary'>
-      <h2>What is CareerAtlas?</h2>
+      <!-- <h2>What is CareerAtlas?</h2> -->
```

Comment:
I left this in just in case we wanted it again. If I delete it, and then add it back in today, will it count as a feature add? @jakerella